conf/config.yaml

```diff
@@ -80,13 +80,13 @@ apisix:
   #   udp:                        # UDP proxy port list
   #     - 9200
   #     - 9211
-  dns_resolver:                   # default DNS resolver, with disable IPv6 and enable local DNS
+  dns_resolver:                   # If not set, read from `/etc/resolv.conf`
```

Comment:
I think it's easier to use extra separate fields to decide whether to use local `/ etc / resolv.conf`.

for example: `dns_resolver_local: true`

Reply:
https://github.com/apache/incubator-apisix/pull/1217/commits/7ec7fd0d8596ea9002398d697f267f4313ba447b#diff-82e3bcd74a1f6e15d0577acef2d4fa8cR88

this commit add a swith for enable local DNS

Reply:
After discussion on the mailing list, the current implementation is better.

https://lists.apache.org/thread.html/r78c7780ae94bb543d448125327c2038ca6bff328946120dd48a2bb5f%40%3Cdev.apisix.apache.org%3E